Model/Config/Source/Terminals.php

```diff
@@ -47,6 +55,7 @@ public function toOptionArray()
                 $terminals[] = ['value' => $terminal->Title, 'label' => $terminal->Title];
             }
         } catch (\Exception $e) {
+            $this->logger->debug('Exception', $e->getMessage());
```

Comment:
If `Exception` is supposed to represent the class name then maybe use get_class($e). The second parameter should be an array and getMessage() returns a string so this code will result in a TypeError and probably halt the program in a fatal state.
Probably this is what you intended instead:
```suggestion
            $this->logger->debug('Exception: ' . $e->getMessage());
```

Reply:
Done